### PR TITLE
IDEM-405

### PR DESCRIPTION
**Pull request by JasminFarhi**
IMG_7773 (1)

### CODE REVIEW

File: app/src/hooks/userVendors.ts [hunk 1]
```diff
@@ -28,7 +28,9 @@ const useVendors = (): Hooks => {
     try {
       const verification = await verificationStorage.get();
       if (!verification) {
-        throw new Error("Verification not found");
+        throw new Error(
+          "Claims not verified. Please verify your claims on the profile page"
+        );
       }
```
Comment (brettvniekerk) [inline app/src/hooks/userVendors.ts:34]: No catch statement?

**Review by xiangzhe0812 — COMMENTED** [review]
Maybe take user to profile page once they click ok

**Comment by bitcoinbrisbane** [comment]
> Maybe take user to profile page once they click ok

I think its ok for this build not to.

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
The required claims have not been verified.  Please verify these claims on the profile page first.

**Review by bitcoinbrisbane — APPROVED** [review]

File: app/src/hooks/userVendors.ts (hunk 1 above)
Comment (brettvniekerk) [reply app/src/hooks/userVendors.ts:34]: Oh lol, found it

**Review by brettvniekerk — APPROVED** [review]

**Pull request merged**